### PR TITLE
Update events.md

### PR DESCRIPTION
**Pull request by tomblench**
See #401 

Update Events documentation to take account of API rewrite:

- Adjust class names and make a few tweaks

### CODE REVIEW

File: doc/events.md [hunk 1]
```diff
@@ -45,8 +44,8 @@ that anonymous inner classes cannot be used as event subscribers.
 Here the `DocumentNotificationClient` is subscribing to the `DocumentCreated` event. Because the
 events are just classes, they obey the type hierarchy. This means that it is also possible to
 subscribe to the generic `DocumentModified` event to be notified of all Created/Deleted/Updated
-events and then use reflection and downcasting if needed, however if you want to listen for multiple 
-event types eg `DocumentCreated` and `DatastoreCreated` using the same subscriber method, the 
+events and then use reflection and downcasting if needed. However if you want to listen for multiple
```
Comment (ricellis) [inline doc/events.md]: I think this needs a comma now
> However,

File: doc/events.md [hunk 2]
```diff
@@ -45,8 +44,8 @@ that anonymous inner classes cannot be used as event subscribers.
 Here the `DocumentNotificationClient` is subscribing to the `DocumentCreated` event. Because the
 events are just classes, they obey the type hierarchy. This means that it is also possible to
 subscribe to the generic `DocumentModified` event to be notified of all Created/Deleted/Updated
-events and then use reflection and downcasting if needed, however if you want to listen for multiple 
-event types eg `DocumentCreated` and `DatastoreCreated` using the same subscriber method, the 
+events and then use reflection and downcasting if needed. However if you want to listen for multiple 
+event types eg `DocumentCreated` and `DocumentStoreCreated` using the same subscriber method, the
```
Comment (ricellis) [inline doc/events.md]: should fix this abbreviation
> e.g.

File: doc/events.md [hunk 3]
```diff
@@ -1,39 +1,38 @@
...
-The `DatastoreManager` class posts events about Databases:
+The `DocumentStore` class posts events about DocumentStores:
 
-* `DatabaseClosed`
-* `DatabaseCreated`
-* `DatabaseDeleted`
-* `DatabaseOpened`.
+* `DocumentStoreClosed`
+* `DocumentStoreCreated`
+* `DocumentStoreDeleted`.
```
Comment (ricellis) [inline doc/events.md]: What about `DocumentStoreOpened`? The class still exists and appears to be posted.

Comment (tomblench) [inline doc/events.md]: You're right!

File: doc/events.md (hunk 1 above)
Comment (tomblench) [inline doc/events.md]: Agreed.

File: doc/events.md (hunk 2 above)
Comment (tomblench) [inline doc/events.md]: We've had discussions about this before and in the absence of a style guide I prefer it without punctuation. We don't write "I.B.M." or "to-day" in modern writing.

[The Guardian](https://www.theguardian.com/guardian-observer-style-guide-e) says "no full points", but it appears that [The Chicago Manual of Style](http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Abbreviations/faq0047.html) does use them.

Comment (ricellis) [inline doc/events.md]: are we agreed on "for example" instead :)

**Comment by tomblench** [comment]
@ricellis comments addressed in 930d2fb37ad5f711c896197cc65c5dab133d497c

**Pull request merged**